### PR TITLE
Adding ftpd logLevel to options and runtime, also adding tests

### PR DESCRIPTION
**Pull request by EricByers**
- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `npm run lint -- package-name` if a `tslint.json` is present.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/sstur/nodeftpd/blob/0d299f78af7f0006a9d5a084614bd8b56a369d1b/lib/FtpServer.js
- [x] Increase the version number in the header if appropriate. - Not needed I believe, since version did not increase.

### CODE REVIEW

File: ftpd/index.d.ts [hunk 1]
```diff
@@ -12,6 +12,14 @@ import fs = require("fs");
 import net = require("net");
 import tls = require("tls");
 
+export enum LogLevel {
```
Comment (rbuckton) [inline ftpd/index.d.ts]: I would mark this as a `const` enum, as there is no exported _value_ named `LogLevel`.

File: ftpd/index.d.ts [hunk 2]
```diff
@@ -195,4 +208,9 @@ export declare class FtpServer extends events.EventEmitter {
      * Stop listening
      */
     public close(callback?: () => void): void;
+
+    /**
+     * Change/Retrieve logLevel at runtime.
+     */
+    public debugging:LogLevel;
```
Comment (rbuckton) [inline ftpd/index.d.ts]: minor nit: Add a space between `:` and `LogLevel`.

**Review by rbuckton — CHANGES_REQUESTED** [review]

File: ftpd/index.d.ts (hunk 1 above)
Comment (EricByers) [inline ftpd/index.d.ts]: Good idea, thanks!  Done.

**Review by rbuckton — APPROVED** [review]

**Comment by rbuckton** [comment]
Thank you for your submission.

**Pull request merged**